docker-compose.yml

```diff
@@ -13,20 +13,20 @@ services:
       - lf-ui-dist:/data/src/dist
 
       # for developer convenience volume map
-      - ../webpack.config.js:/data/webpack.config.js
-      - ../webpack-dev.config.js:/data/webpack-dev.config.js
-      - ../webpack-prd.config.js:/data/webpack-prd.config.js
-      - ../package.json:/data/package.json
-      - ../package-lock.json:/data/package-lock.json
+      - ./webpack.config.js:/data/webpack.config.js
+      - ./webpack-dev.config.js:/data/webpack-dev.config.js
+      - ./webpack-prd.config.js:/data/webpack-prd.config.js
+      - ./package.json:/data/package.json
+      - ./package-lock.json:/data/package-lock.json
 
       # needed these volume maps so changes to typescript/scss would be reflected in running app, actually rebundled and output to dist which is then shared to the app container.
-      - ../src/angular-app:/data/src/angular-app
-      - ../src/sass:/data/src/sass
-      - ../src/Site/views/languageforge/theme/default/sass:/data/src/Site/views/languageforge/theme/default/sass
+      - ./src/angular-app:/data/src/angular-app
+      - ./src/sass:/data/src/sass
+      - ./src/Site/views/languageforge/theme/default/sass:/data/src/Site/views/languageforge/theme/default/sass
 
   app:
     build:
-      context: ..
+      context: .
```

Comment:
did you check whether this is actually required now?  I might expect that to be the default.

Reply:
Using simply: `build: path/to/Dockerfile-folder` seems to set the context to that folder, which causes lots of trouble in the `Dockerfile`s, so I'm just going to remove the meaningless `context` properties as suggested here.